cms/api.py

```diff
@@ -204,7 +204,7 @@ def create_title(language, title, page, menu_title=None, slug=None,
                  created_by='python-api', soft_root=False, in_navigation=False,
                  template=TEMPLATE_INHERITANCE_MAGIC,
                  limit_visibility_in_menu=constants.VISIBILITY_ALL,
-                 xframe_options=constants.X_FRAME_OPTIONS_INHERIT):
+                 xframe_options=constants.X_FRAME_OPTIONS_INHERIT):  # TODO: Deprecate this
```

Comment:
It makes this much easier without changing the create_title api as you have done. We will need a plan of how we could replace this in the future, it will affect all internal tests, and if we have 2 at the same time the tests would need to cover both too. Very ugly. 

This was left alone due to the sheer amount of changes that would be required to every other package and every other packages test suite.